jha/src/views/enrolment/FPCareEligibilityPage.vue

```diff
@@ -190,6 +192,23 @@ export default {
       scrollTo(0);
     }
   },
+  computed: {
+    msgCode(){
+      if (this.applyFPC === 'N') {
+        // Not applying for FPC
+        return 0;
```

Comment:
Instead of inline values for this, can you use a constant with a single source of truth?

Similar to the document types. Something like:
```javascript
export const EQFPCMsgCode = {
  NotApplyingForFPC: 0,
  NotMeetsCriteria: 1,
  ...
};
```
That way each case can be easily identified in the templates that use these.

Reply:
Done, please re-review